docs/user-guide/commands/argocd_app_sync.md

```diff
@@ -59,6 +59,7 @@ argocd app sync [APPNAME... | -l selector | --project project-name] [flags]
       --prune                                             Allow deleting unexpected resources
       --replace                                           Use a kubectl create/replace instead apply
       --resource stringArray                              Sync only specific resources as GROUP:KIND:NAME or !GROUP:KIND:NAME. Fields may be blank and '*' can be used. This option may be specified repeatedly
+      --respect-ignore-differences                        A configuration option to ensure difference between live and desired states of a resource are ignored during synchronization
```

Comment:
The description is confusing, making it looks like the difference is always ignored.